inductiva/molecules/scenarios/protein_solvation/README.md

```diff
@@ -0,0 +1,44 @@
+# Protein Solvation Scenario
+
+This scenario models the dynamics of a protein solvated in water inside a virtual cubic box. The system is modelled with the motion equations for each individual atoms according to the prescription of a molecular force field approximation.
+
+To initilaize the scenario, the user can define the following parameters:
+-- Protein .pdb file: structural .pdb file of the protein to simulate;
+-- Temperature of the system in K.
+
+Now, the user is ready to simulate the protein dynamics. Note that the scenario currently supports only protein .pdb files, like the ones available in AlphaFold. RCSB's protein files can be provided to this scenario, but all the non-proteic parts of the system like ligands, non-standard residues or water molecules will be stripped before simulation.
```

Comment:
suggestion: Now, the user is prepared to simulate protein dynamics. Please note that the scenario currently supports only PDB files as input. We have conducted extensive testing with structures generated by the AlphaFold2 model and structures from the RCSB database. Before simulation, files undergo pre-processing to strip away all non-protein components of the system, such as ligands, non-standard residues, or water molecules, leaving only the protein.